frontend/javascripts/viewer/model/sagas/volume/proofread_saga.ts

```diff
@@ -586,6 +621,166 @@ function* performMinCut(
   return false;
 }
 
+function* performPartitionedMinCut(_action: MinCutPartitionsAction | EnterAction): Saga<void> {
```

Comment:
This has quite some duplication with the other code. Do you think it makes sense to deduplicate here or? Because at some code lines it diverges. I could try to extract common methods like already done with `prepareSplitOrMerge`

Reply:
If you see a big chunk of code that is duplicated please go ahead (I didn't spot one immediately), but otherwise it's fine.